master/static/srv/get_models.sql

```diff
@@ -4,8 +4,9 @@ LEFT JOIN users as u ON u.id = m.user_id
 WHERE ($1 = 0 OR m.id = $1)
 AND ($2 = '' OR m.archived = $2::BOOL)
 AND ($3 = '' OR (u.username IN (SELECT unnest(string_to_array($3, ',')))))
-AND ($4 = '' OR (m.labels <@ string_to_array($4, ',')))
-AND ($5 = '' OR m.name ILIKE $5)
-AND ($6 = '' OR m.description ILIKE $6)
+AND ($4 = '' OR m.user_id IN (SELECT unnest(string_to_array($4, ',')::int [])))
+AND ($5 = '' OR (m.labels <@ string_to_array($5, ',')))
+AND ($6 = '' OR m.name ILIKE $6))
```

Comment:
I think the issue is just the double parens here: ))

Reply:
looks like that was it, thanks for catching that!